app/views/podcast_episodes/_episodes_feed.html.erb

```diff
@@ -19,6 +19,12 @@
       <h4>
         <a href="/<%= episode.podcast.slug %>">
           <%= episode.podcast.title %>
+          <% if episode.published_at.present? %>
+            ・<time class="published-at" datetime="<%= episode.decorate.published_timestamp %>">
+            <%= episode.decorate.readable_publish_date %>
```

Comment:
you don't need to call `.decorate`, the collection has been already decorated in the controller

Reply:
You will also need to decorate the collection inside the `StoriesController#handle_podcast_index`

Reply:
thanks, i'm working on it